e2e/testing/chain.go

```diff
@@ -253,18 +265,20 @@ func NewTestChain(t *testing.T, chainIdx int, opts ...interface{}) *TestChain {
 		app: archApp,
 		curHeader: tmProto.Header{
 			ChainID: chainid,
+			Height:  1,
 			Time:    time.Unix(0, 0).UTC(),
 		},
-		txConfig:    encCfg.TxConfig,
+		txConfig:    archApp.TxConfig(),
 		valSet:      validatorSet,
 		valSigners:  valSigners,
 		accPrivKeys: genAccPrivKeys,
 	}
-	chain.BeginBlock()
```

Comment:
What is happening between lines 263-281;
Copypasta?

Reply:
this will need to be handled when the tests are further fixed. Didnt want to lose the original code by removing it.